content/en/docs/instrumentation/java/manual.md

```diff
@@ -35,6 +35,87 @@ the most important piece of telemetry source-identifying info.
 
 For example:
 
+<details>
+  <summary>Maven top level dependencies. Click me.</summary>
+  Note that opentelemetry-semconv does not exist (yet) as 1.20.1 but only as 1.20.1-alpha.
+  
+  ```
+  
+  <project>
+      <dependencyManagement>
+          <dependencies>
+              <dependency>
+                  <groupId>io.opentelemetry</groupId>
+                  <artifactId>opentelemetry-bom</artifactId>
+                  <version>1.20.1</version>
+                  <type>pom</type>
+                  <scope>import</scope>
+              </dependency>
+          </dependencies>
+      </dependencyManagement>
+      
+      <dependencies>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-api</artifactId>
+          </dependency>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-sdk</artifactId>
+          </dependency>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-sdk-trace</artifactId>
+          </dependency>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-sdk-metrics</artifactId>
+          </dependency>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-sdk-common</artifactId>
+          </dependency>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-exporter-otlp</artifactId>
+          </dependency>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-context</artifactId>
+          </dependency>
```

Comment:
`opentelemetry-api` includes `opentelemetry-context`, there should be no need to add that dependency manually. 
```suggestion
```

Reply:
Regarding the dependencies listed here.

I refer you to https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html
and this comment "Maven also provides [dependency:analyze](https://maven.apache.org/plugins/maven-dependency-plugin/analyze-mojo.html) plugin goal for analyzing the dependencies: it helps making this best practice more achievable."

This is exactly what I did. Your maven file should include all the dependencies which materialize as an "import" in the Java code.
I explicitly ran `mvn dependency:analyze` to see what was missing or extra. If you have never used it, please give it a try.
Then the list present is complete, nothing missing, nothing extra.

Reply:
I think the scenario described in that article is exceptionally rare for opentelemetry and unlikely to ever pass. Not only are breaking changes like that simply not an acceptable outcome for the project, but context in particular is such a foundational building block for the API that removing it would be akin to removing the notion of arithmetic in a programming language.

Reply:
hey @odupuy! I think in these docs it's beneficial to keep things simple and not overwhelm new users with the modular breakdown of the API.

Experienced maven users can always run `dependency:analyze` and add the transitive dependencies if they are familiar with / follow this practice.

Reply:
Yep, the intention here is to have a minimal maven/gradle example that users can copy-paste to get started with OpenTelemetry. `dependency:analyze` and pinning the transitive dep versions are exercises left to the reader.